service.clocknotify/addon.xml

```diff
@@ -0,0 +1,18 @@
+﻿<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="service.clocknotify"
+    name="Clock Notify" version="1.0.1" provider-name="tuoxin">
+  <requires>
+    <import addon="xbmc.python" version="2.24.0"/>
```

Comment:
Python version is jarvis, but your PR is for Krypton

Reply:
Transparent Icons aren't allowed, Could you please change it?

Reply:
@Lunatixz just jumping in here for a question before I submit my own add-on. How do I make it compatible with lower versions if we need to raise PR against Krypton branch? My Add-on works with Isengard(2.20.0 in Add-on XML) so bit confused about how to do it?

Reply:
You can leave a lower python api, however Kodi users will not have access to it. If you push to the Krypton branch only Krypton users can see and download the plugin.